File: pixo-c#/DalSql/models/Order.cs
using System;
using System.Collections.Generic;

namespace DalSql.models;

public partial class Order
{
    public int Id { get; set; }

    public int? CustomerId { get; set; }

    public DateTime? OrderDate { get; set; }

    public decimal? TotalPrice { get; set; }

    public virtual Customer? Customer { get; set; }

    public virtual ICollection<OrderItem> OrderItems { get; set; } = new List<OrderItem>();
}

File: projectAngular/src/app/models/order.ts
export class Order {
  id: number;             // מזהה ההזמנה
  customersId: number;    // מזהה הלקוח
  orderDate: Date;        // תאריך ההזמנה
  totalPrice: number;     // מחיר כולל

  constructor(id: number,customersId: number,orderDate: Date,totalPrice: number) {
    this.id = id;
    this.customersId = customersId;
    this.orderDate = orderDate;
    this.totalPrice = totalPrice;
  }
}

const order = new Order(1, 1, new Date(), 100.50);
